br/tests/br_full_cluster_restore/run.sh

```
@@ -128,6 +129,8 @@ run_sql_as user2 "123456" "select count(*) from db2.t1" || true
 check_contains "SELECT command denied to user"
 # user3 can only query db1.t1 using ssl
 # ci env uses mariadb client, ssl flag is different with mysql client
+run_sql "select * from mysql.global_priv;"
+cat $TEST_DIR/sql_res.$TEST_NAME.txt
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)